### DISCUSSION TITLE
eglot-dired-mode?

### DISCUSSION THREAD

**Discussion by guillaumebrunerie** [joaotavora/eglot#1168] [Q&A] [answered]
`lsp-mode` includes `lsp-dired-mode` which is basically `dired-mode` but showing which files have errors/warnings (diagnostics). It's pretty handy to have a quick overview of all the files that need to be fixed. Is there something similar/equivalent in Eglot?

**Comment by joaotavora** [comment, marked as answer]
`M-x flymake-show-project-diagnostics` is a thing.  Maybe it works with your project/lsp server. Works for `clangd`: 

![image](https://user-images.githubusercontent.com/387011/218608587-de9e3d70-7c41-4f11-a5cd-df86b22908f4.png)
